wolfssl/tests/ping_pong.rs

```diff
@@ -192,18 +193,27 @@ async fn server<S: SockIO>(sock: S, protocol: Protocol) {
     println!("[Server] Finished");
 }
 
-#[tokio::test]
-async fn dtls() {
-    use Protocol::*;
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_3 => panics; "client_1.2_server_1.3")]
```

Comment:
The `panics` matcher takes an optional string and asserts that it appears in the panic message.

I've been tripped up in the past by code which panicked for some other reason than the one we were expecting, so adding `"record layer version error"` (or whatever unique-ish string there is) might be wise for these cases (and the TLS ones)

Reply:
Nice suggestion. Will incorporate it